platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/AbstractPKIHandler.java

```diff
@@ -103,7 +103,13 @@ public HandlerResult getNormalizedToken(
     } else {
       String errorMsg = "The certificate used to complete the request has been revoked.";
       LOGGER.info(errorMsg);
-      throw new ServletException(errorMsg);
+      try {
+        httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "Your certificate is revoked.");
```

Comment:
@shaundmorris Is this a false positive? We have the code catching a generic `Exception` already

Reply:
It is a false positive. Typically `null` conditions are checked for null rather than catching the null pointer exception. That is probably what the  static checker is looking for. If you wanted to satisfy the checker, you could handle the `null` explicitly:
```
if (httpResponse == null) {
        LOGGER.error("Error returning revoked certificate request because the HTTP response object is invalid.");
}
else {
   try {
        httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "Your certificate is revoked.");
        httpResponse.flushBuffer();
      } catch (Exception e) {
        LOGGER.error("Error returning revoked certificate request.");
      }
}
```